Source/Core/Common/Config/Layer.cpp

```diff
@@ -8,10 +8,47 @@
 
 #include "Common/Config/Config.h"
 #include "Common/Config/Layer.h"
-#include "Common/Config/Section.h"
 
 namespace Config
 {
+namespace detail
+{
+std::string ValueToString(u16 value)
```

Comment:
geez, looking at this, our encoding/format of types is really silly, looking at the list like this (we can't auto-handle the whole range of integral types/floating-point types without special-casing).